index1.html

```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>My Portfolio</title>
+  <!-- stylesheet link  -->
+  <link rel="stylesheet" href="style1.css" />
+  <!-- font awesome -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+
+<body>
+  <header id="navigation">
+    <a href="#" class="logo">My Logo</a>
+    <nav>
+      <div class="mobile-menu">
+        <div class="menu-bar">
+          <i class="fa fa-bars fa-1x hamburger"></i>
+        </div>
+      </div>
+    </nav>
+  </header>
+
+  <section class="hero-section">
+    <div class="hero-container">
+      <h1 id="name">I’m ojwang Briton. Glad to see you!</h1>
+      <p class="details">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact
+        me.
+      </p>
+      <a href="#" class="connect">LET'S CONNECT</a>
+      <ul class="socials">
+        <li>
+          <a href="">
+            <i><img class="icons" src="/images1/twitter.svg" alt="" /></i>
+          </a>
+        </li>
+        <li>
+          <a href="">
+            <i><img class="icons" src="/images1/Linkedin.svg" alt="" /></i>
+          </a>
+        </li>
+        <li>
+          <a href="">
+            <i><img class="icons" src="/images1/github.svg" alt="" /></i>
+          </a>
```

Comment:
- [OPTIONAL ] I can see that you did a great work so far nice design, but here are your icons which are not diplaying , this might be a wrong path , kindly check your icons' path and fix this :point_down:  for your project to look more professional and attractive :wink: 
![Screenshot from 2023-05-05 22-29-44](https://user-images.githubusercontent.com/88894525/236565001-901608fa-51b4-49a8-a5e4-78439b2ca2cd.png)